sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -256,7 +256,7 @@ export default function CompReadyTestReport(props) {
       // a buffer of 1 week after GA.
       const twentyWeeksPreGA = new Date(gaDate.getTime())
       twentyWeeksPreGA.setDate(twentyWeeksPreGA.getDate() - 20 * 7)
-      gaDate.setDate(gaDate.getDate() + 7)
+      gaDate.setDate(gaDate.getDate())
```

Comment:
The problem is 4 weeks before GA always shows up as 5 weeks which isn't right, what's the added buffer for? I'm probably not understanding correctly, can you give an example where it is needed?

Reply:
I can also just move it to a separate PR, this isn't really related to the back button

Reply:
I added the `buffer` so that we could round up or down to the next or previous week.  But 7 doesn't work out so well (e.g., 28 days, it shows 5 weeks which is clearly off).

The `buffer` of 0 works out better.

I ran this to test it:

```js
const fromDate = "2023-04-17T00:00:00Z";
const toDate = "2023-05-16T23:59:59Z";
const version = "4.13";
const buffer = 0;
console.log("buffer: ", buffer);
for (let i = -7; i <= 8; i++) {
  const currentDate = new Date(fromDate);
  currentDate.setDate(currentDate.getDate() + i);

  const val = getSummaryDate2(currentDate, toDate, version, versions, buffer);
  const daysBetween = Math.floor(
    (new Date(toDate) - currentDate) / (1000 * 60 * 60 * 24)
  );

  console.log(`Days Between: ${daysBetween}, Value (val): ${val}`);
}
```

sample output with no buffer:

```
buffer:  0
Days Between: 36, Value (val): About 5 week(s) before '4.13' GA date
Days Between: 35, Value (val): About 5 week(s) before '4.13' GA date
Days Between: 34, Value (val): About 5 week(s) before '4.13' GA date
Days Between: 33, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 32, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 31, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 30, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 29, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 28, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 27, Value (val): About 4 week(s) before '4.13' GA date
Days Between: 26, Value (val): About 3 week(s) before '4.13' GA date
Days Between: 25, Value (val): About 3 week(s) before '4.13' GA date
Days Between: 24, Value (val): About 3 week(s) before '4.13' GA date
Days Between: 23, Value (val): About 3 week(s) before '4.13' GA date
Days Between: 22, Value (val): About 3 week(s) before '4.13' GA date
Days Between: 21, Value (val): About 3 week(s) before '4.13' GA date
```

Reply:
Ok, I'll remove this for now and deal with it separately